doc/classes/EditorInspector.xml

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A tab used to edit properties of the selected node.
+		A control used to edit properties of an object.
 	</brief_description>
 	<description>
-		The editor inspector is by default located on the right-hand side of the editor. It's used to edit the properties of the selected node. For example, you can select a node such as the Sprite2D then edit its transform through the inspector tool. The editor inspector is an essential tool in the game development workflow.
-		[b]Note:[/b] This class shouldn't be instantiated directly. Instead, access the singleton using [method EditorInterface.get_inspector].
```

Comment:
Is this note still relevant, or has the ability to create EditorInspector nodes been added?

Reply:
This note assumes that `EditorInspector` is the Inspector dock, so it says the user should not try to create a singleton. (But it's actually not.)

`EditorInspector` node can be created & added, but there is not enough methods exposed to scripting to make it usable. e.g. `edit(object)` is not exposed so you can't really make the newly created `EditorInspector` inspecting anything.

I think `EditorInspector` could eventually be tweaked to make it safe to be used by the user :)